src-built-in/components/yesNoDialog/src/app.jsx

```diff
@@ -6,6 +6,7 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "../../assets/css/finsemble.css";
 import { FinsembleDialog, FinsembleDialogQuestion, FinsembleDialogButton } from "@chartiq/finsemble-react-controls";
+import Timer from "./timer";
```

Comment:
@carterbs I guess the timer is a useful thing to have in the yes/no dialog. My original plan had been to implement the restart alert as a notification but I don't yet know whether they sufficiently support the timeouts necessary.

@jimbunting what's your thought on whether the restarting alert message should be dialog or notification?

Reply:
The advantage of the yes/no dialog is that the user has the ability to cancel the restart. However, it forces the user to stop what they're doing. 

The advantage of the notification would be that it's less obtrusive and allows the user to keep working. If we could add two buttons to the notifications allowing for "restart now" and "cancel", this would seem optimal.

Reply:
Getting notified that Finsemble is about to restart *should* stop you from working. You don't want to begin something critical, then lose your progress / be unsure of how far you got. I would like a dialog to force the user to make a decision: (cancel/restart now).

https://chartiq.kanbanize.com/ctrl_board/18/cards/6823/details